### DISCUSSION TITLE
Default zoom doesn't work

### DISCUSSION THREAD

**Discussion by extracampine** [sumatrapdfreader/sumatrapdf#2421] [Q&A]
I have the following in the SumatraPDF-settings.txt file:

DefaultZoom = 50

But PDFs still open full-screen. Is this a bug or am I doing something wrong?

**Comment by GitHubRulesOK** [comment]
The main settings include several option on a dropdown for desired default for first opening, 
my default of fit page is probably as installed but you can set yours to 50 if you prefer. 

Note also that if you have remember settings for each document **it will use the last value when closed cleanly for that document** on next reopening.
